lib/spack/spack/test/llnl/util/lock.py

```diff
@@ -1326,6 +1391,8 @@ def test_downgrade_write_fails(tmpdir):
                          [(errno.EACCES, "Fake EACCES error"),
                           (errno.EAGAIN, "Fake EAGAIN error"),
                           (errno.ENOENT, "Fake ENOENT error")])
+@pytest.mark.skipif(_platform == 'win32',
+                    reason='not supported on windows')
 def test_poll_lock_exception(tmpdir, monkeypatch, err_num, err_msg):
```

Comment:
As I noted in a couple other testing modules, since all these tests are disabled, can we replace the per-test `skip` statements with a single module-level `skip`?

Also, would it be possible to set `locks: false` in the Windows config file, and to move all locking changes to another PR (which we could revisit after merging this one)?

Reply:
Correct, all tests here can/will be skipped at the module level. That wouldn't be a problem. We don't necessarily need to factor out any changes currently in this branch, as we can generally confirm that nothing is damaged on non-Windows systems.

Reply:
Done in second review PR.